.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: java
-
+sudo: false
+dist: trusty
```

Comment:
I've must add that because build have errored: https://travis-ci.org/vavr-io/vavr-beanvalidation2/builds/566001535

Reply:
actually we should add openjdk11. I will do that

Reply:
sigh, travis build fails due to jacoco. I will probably have to talk to daniel about this. will do later